doc/material-migration.md

```diff
@@ -404,4 +408,4 @@ ToggleButton|MaterialToggleButtonIconStyle|MaterialIconToggleButtonStyle|
 Namespace|Previous Package|New Package|
 -|-|-
 ControlExtensions.Icon|Uno.Material.Extensions|Uno.Material|
-Ripple|Uno.Material.Controls|Uno.Material|
\ No newline at end of file
+Ripple|Uno.Material.Controls|Uno.Material|
```

Comment:
@kazo0 don't think a change here is needed no?

Reply:
I didn't change anything there, that line was just deleted and added again while I made changes on other lines
The code is still the same